src/test/java/com/puppycrawl/tools/checkstyle/JavadocPropertiesGeneratorTest.java

```diff
@@ -238,9 +238,9 @@ public void testHelp(@SysErr Capturable systemErr, @SysOut Capturable systemOut)
     @Test
     public void testJavadocParseError() throws Exception {
         final String path = getPath("InputJavadocPropertiesGeneratorJavadocParseError.java");
+        final String destinationAbsolutePath = DESTFILE.getAbsolutePath();
```

Comment:
As I see, this expression `DESTFILE.getAbsolutePath()` is repeated 10 times in the file.
Please extract it to class constant and reuse everywhere.
It will also help to avoid such sonar issues in future.

Reply:
Interesting fact that sonar do not complain about them :), sonar complained about 1, I fixed few, but there are even more .

Reply:
done.